docker/Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN pip install cython
 RUN pip install xtcocotools
 
 # Install MMCV
-RUN pip install mmcv-full==latest+torch1.6.0+cu101 -f https://download.openmmlab.com/mmcv/dist/index.html
+RUN pip install --no-cache-dir --upgrade pip wheel setuptools
+RUN pip install --no-cache-dir mmcv-full==1.3.17 -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6.0/index.html
```

Comment:
What about using `-U mmcv-full` instead of specifying the version?

Reply:
It may report an error，because downstream libraries may have version restrictions。
e.g.：
when I import mmpose, there appears "AssertionError: MMCV==1.3.5 is used but incompatible. Please install mmcv>=1.3.8, <=1.5.0"

Reply:
Sure, we can merge this pr as it is now.  We will check if there is a better way to install the latest mmcv in docker in the future. Thank you.